packages/mobile/src/screens/edit-track-screen/EditTrackScreen.tsx

```diff
@@ -0,0 +1,48 @@
+import type { UploadTrack } from '@audius/common'
+import { Formik } from 'formik'
+import * as Yup from 'yup'
+
+import { EditTrackNavigator } from './EditTrackNavigator'
+import type { FormValues, EditTrackScreenProps } from './types'
+
+const EditTrackSchema = Yup.object().shape({
```

Comment:
we do custom validations in so many places. footnote to remember to try to consolidate some of this